classes/locallib/bigbluebutton.php

```diff
@@ -47,12 +47,13 @@ class bigbluebutton {
      */
     public static function action_url($action = '', $data = array(), $metadata = array()) {
         $baseurl = self::sanitized_url() . $action . '?';
-        $params = '';
-        foreach ($data as $key => $value) {
-            $params .= '&' . $key . '=' . urlencode($value);
-        }
-        foreach ($metadata as $key => $value) {
-            $params .= '&' . 'meta_' . $key.'=' . urlencode($value);
+        $params = http_build_query($data, '', '&');
+        if(!empty($metadata)){
```

Comment:
codechecker is going to complain for the space after the if and before the {

also, can this be done without adding (even more :)) cyclic complexity. Scrutinizer is also complaining.